guide/guide.md

```diff
@@ -25,7 +25,7 @@ typically by using a formatting tool's default settings.
 ### Indentation and line width
 
 * Use spaces, not tabs.
```

Comment:
I saw this initially, I expects `4 spaces` here but I skipped through the next line.